recipes/r-soupx/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+{% set name = "SoupX" %}
+{% set version = "1.4.5" %}
+
+package:
+  name: r-soupx
+  version: {{ version }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/{{ name }}_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
+  sha256: 237eb83ca384dff1f26b8c0bcbafec6389bbfc9abdccf84c1783144836708c4f
+
+build:
+  merge_build_host: true  # [win]
+  noarch: generic
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip      # [win]
+  host:
+    - r-base
+    - r-matrix
+    - r-ggplot2
+    - r-seurat
+  run:
+    - r-base
+    - r-matrix
+    - r-ggplot2
+    - r-seurat
+
+test:
+  commands:
+    - $R -e "library('"{{ name }}"')"    # [not win]
+    - "\"%R%\" -e \"library('SoupX')\""  # [win]
+
+about:
+  home: https://github.com/constantAmateur/SoupX/
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```

Comment:
Seems there's a copy/paste error here.

While the recipe is fine, we generally advice to use https://github.com/bgruening/conda_r_skeleton_helper for generating conda-forge R packages.

Reply:
Thanks! I thought `conda skeleton cran <package>` was for bioconda only and was following https://conda-forge.org/#add_recipe, which seems to suggest copying/modifying the repo's example recipe.

I've now re-generated it with `conda skeleton`.

Reply:
Could you review it again please? @cbrueffer

Reply:
Please add conda-forge/r to maintainers. The recipe looks OK otherwise; what I meant earlier was actually using the script at https://github.com/bgruening/conda_r_skeleton_helper for recipe generation; it runs conda skeleton cran and cleans up the recipe from there (such adding conda-forge/r, removing comments, etc).

Reply:
I see, thank you! conda-forge/r added to maintainers and comments removed.